gematria/datasets/python/annotating_importer_test.py

```diff
@@ -0,0 +1,147 @@
+            assembly="\timull\t%edx, %edx",
+            address=18446744073709547789,
+            machine_code=b"\017\257\322",
+        ),
+        basic_block_pb2.MachineInstructionProto(
+            assembly="\taddl\t%edx, %eax",
+            address=18446744073709547792,
+            machine_code=b"\001\320",
+        ),
+        basic_block_pb2.MachineInstructionProto(
+            assembly="\tdecl\t%ecx",
+            address=18446744073709547794,
+            machine_code=b"\377\311",
+        ),
+    ),
+    canonicalized_instructions=(
+        _CanonicalizedInstructionProto(
+            mnemonic="MOV",
+            llvm_mnemonic="MOV32rr",
+            output_operands=(_CanonicalizedOperandProto(register_name="EDX"),),
+            input_operands=(_CanonicalizedOperandProto(register_name="ECX"),),
+        ),
+        _CanonicalizedInstructionProto(
+            mnemonic="IMUL",
+            llvm_mnemonic="IMUL32rr",
+            output_operands=(_CanonicalizedOperandProto(register_name="EDX"),),
+            input_operands=(
+                _CanonicalizedOperandProto(register_name="EDX"),
+                _CanonicalizedOperandProto(register_name="EDX"),
+            ),
+            implicit_output_operands=(
+                _CanonicalizedOperandProto(register_name="EFLAGS"),
+            ),
+        ),
+        _CanonicalizedInstructionProto(
+            mnemonic="ADD",
+            llvm_mnemonic="ADD32rr",
+            output_operands=(_CanonicalizedOperandProto(register_name="EAX"),),
+            input_operands=(
+                _CanonicalizedOperandProto(register_name="EAX"),
+                _CanonicalizedOperandProto(register_name="EDX"),
+            ),
+            implicit_output_operands=(
+                _CanonicalizedOperandProto(register_name="EFLAGS"),
+            ),
+        ),
+        _CanonicalizedInstructionProto(
+            mnemonic="DEC",
+            llvm_mnemonic="DEC32r",
+            output_operands=(_CanonicalizedOperandProto(register_name="ECX"),),
+            input_operands=(_CanonicalizedOperandProto(register_name="ECX"),),
+            implicit_output_operands=(
+                _CanonicalizedOperandProto(register_name="EFLAGS"),
+            ),
+        ),
+    ),
+)
+
+
+class AnnotatingImporterTest(absltest.TestCase):
+  _ELF_OBJECT_FILEPATH = (
+      r"com_google_gematria/gematria/testing/testdata/simple_x86_elf_object"
+  )
+  _PERF_DATA_FILEPATH = (
+      r"com_google_gematria/gematria/testing/testdata/"
+      r"simple_x86_elf_object.perf.data"
+  )
+  _SOURCE_NAME = "test: skl"
+
+  def setUp(self):
+    super().setUp()
+
+    self._x86_llvm = llvm_architecture_support.LlvmArchitectureSupport.x86_64()
+    self._x86_canonicalizer = canonicalizer.Canonicalizer.x86_64(self._x86_llvm)
+
+    self._runfiles_dir = os.environ.get("PYTHON_RUNFILES")
+    self._runfiles_env = runfiles.Create({"RUNFILES_DIR": self._runfiles_dir})
+    assert self._runfiles_env is not None
```

Comment:
I'd prefer an explicit self.assertIsNotNone(self._runfiles_env).

That should always work, assert statements and all code in them can be disabled by a flag of the runtime.